x-pack/plugins/serverless_search/public/application/components/connectors/elastic_managed_connector_coming_soon.tsx

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useEffect } from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiIcon,
+  EuiTitle,
+  EuiText,
+  EuiButton,
+  EuiToolTip,
+  EuiBadge,
+  EuiStepsHorizontal,
+  EuiStepsHorizontalProps,
+  EuiButtonEmpty,
+  EuiFormRow,
+  EuiTextArea,
+  EuiLink,
+  EuiForm,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/react';
+import { useKibanaServices } from '../../hooks/use_kibana';
+import { useConnectorTypes } from '../../hooks/api/use_connector_types';
+import { useAssetBasePath } from '../../hooks/use_asset_base_path';
+
+import { BACK_LABEL } from '../../../../common/i18n_string';
+
+export const ElasticManagedConnectorCommingSoon: React.FC = () => {
+  const connectorTypes = useConnectorTypes();
+  const allConnectors = connectorTypes ? connectorTypes.map((connectorType) => connectorType) : [];
+
+  const getRandomConnectors = (connectors: typeof allConnectors, count: number) => {
+    const shuffled = connectors.sort(() => 0.5 - Math.random());
+    return shuffled.slice(0, count);
+  };
+
+  const [randomConnectors, setRandomConnectors] = useState(() =>
+    getRandomConnectors(allConnectors, 4)
+  );
+
+  useEffect(() => {
```

Comment:
Also, given that you're using this logic in multiple places now, you should extract this to a `hook` to re-use.

Reply:
This random animation was not well received, so that with this [commit](https://github.com/elastic/kibana/pull/196338/commits/a400f7c5fa1729ab87a71fadbd05e606f664e3b8) it just displays static connector icons